docs/getting-started/raster/raster_step-by-step.md

```diff
@@ -35,13 +35,40 @@ flowchart LR
     C -- layer_params --> D
 ```
 
-
 ## Query CSW catalog (Step 1)
 Query **RASTER CSW catalog** service to find item(s) according to desired filter [example are here](/catalog-information/query-examples.md).
+</br></br>
```

Comment:
try to keep MD clean from HTML as much as possible

Reply:
fixed